File: src/test/java/issubsequence/SolutionTest.java
package issubsequence;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class SolutionTest {

    private final Solution solution = new Solution();

    @Test
    public void testOne() {
        String s = "abc";
        String t = "ahbgdc";

        boolean subsequence = solution.isSubsequence(s, t);

        assertThat(subsequence).isTrue();
    }

    @Test
    public void testTwo() {
        String s = "axc";
        String t = "ahbgdc";

        boolean subsequence = solution.isSubsequence(s, t);

        assertThat(subsequence).isFalse();
    }

}

File: src/test/kotlin/dev/sanket/sametree/SolutionTest.kt
package dev.sanket.sametree

import assertk.assertThat
import assertk.assertions.isTrue
import dev.sanket.dev.sanket.common.buildTree
import dev.sanket.dev.sanket.sametree.Solution
import org.junit.jupiter.api.Test

class SolutionTest {

    private val solution = Solution()

    @Test
    fun testOne() {
        val p = buildTree(intArrayOf(1, 2, 3), 0)
        val q = buildTree(intArrayOf(1, 2, 3), 0)

        assertThat(solution.isSameTree(p, q)).isTrue()
    }
}